src/commands/build.rs

```diff
@@ -180,6 +187,8 @@ pub async fn build(
         })
         .collect::<Result<Vec<_>>>()?;
 
+    drop(loading_span_guard);
```

Comment:
Note: A few lines below is an `info!` where we probably want to add `parent: &loading_span`?

Reply:
Right.